certora/harness/MarketplaceHarness.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.23;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IGroth16Verifier} from "../../contracts/Groth16.sol";
+import {MarketplaceConfig} from "../../contracts/Configuration.sol";
+import {Marketplace} from "../../contracts/Marketplace.sol";
+import {RequestId} from "../../contracts/Requests.sol";
+
+contract MarketplaceHarness is Marketplace {
+    constructor(MarketplaceConfig memory config, IERC20 token, IGroth16Verifier verifier) Marketplace(config, token, verifier) {}
+
+    function requestContext(RequestId requestId) public returns (Marketplace.RequestContext memory) {
+        return _requestContexts[requestId];
+    }
```

Comment:
Atm this harness is used instead of the actual contract, so we get access to `_requestContexts`.

Looking into solving this with `ghost` variables instead so we don't have to add this harness and change the property's visibility.